snapcraft.yaml

```diff
@@ -3,7 +3,7 @@ summary: Create Ubuntu images
 description: |
   Official tool for building Ubuntu images, currently supporing Ubuntu Core
   snap-based images and preinstalled Ubuntu classic images.
-version: 3.0+snap1~beta8
+version: 3.0+snap1~beta11
```

Comment:
Is there a reason you don't use "version: git" or something like that? (not a very important question!)

Reply:
Historically the version was `1.x+snap<y>`. Not sure why we started using this convention, but I could change it if you'd like.